inc/options/browsercache.php

```diff
@@ -1,449 +1,652 @@
-						<?php Util_Ui::sealing_disabled( 'browsercache.' ) ?>
-						name="browsercache__no404wp__exceptions" cols="40" rows="5"><?php echo esc_textarea( implode( "\r\n", $this->_config->get_array( 'browsercache.no404wp.exceptions' ) ) ) ?></textarea>
-					<p class="description"><?php _e( 'Never process 404 (not found) events for the specified <acronym title="Uniform Resource Identifier">URI</acronym>s.', 'w3-total-cache' ) ?></p>
+						<?php Util_Ui::sealing_disabled( 'browsercache.' ); ?>
+						name="browsercache__no404wp__exceptions" cols="40" rows="5"><?php echo esc_textarea( implode( "\r\n", $this->_config->get_array( 'browsercache.no404wp.exceptions' ) ) ); ?></textarea>
+					<p class="description">
+						<?php
+						echo wp_kses(
+							sprintf(
+								// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag.
+								__(
+									'Never process 404 (not found) events for the specified %1$sURI</acronym>s.',
+									'w3-total-cache'
+								),
+								'<acronym title="' . __( 'Uniform Resource Identifier', 'w3-total-cache' ) . '">',
+								'</acronym>'
+							),
+							array(
+								'acronym' => array(
+									'title' => array(),
+								),
+							)
+						);
+						?>
+					</p>
 				</td>
 			</tr>
 			<?php
-			Util_Ui::config_item( array(
-					'key' => 'browsercache.rewrite',
-					'disabled' => Util_Ui::sealing_disabled( 'browsercache.' ),
-					'control' => 'checkbox',
+			Util_Ui::config_item(
+				array(
+					'key'            => 'browsercache.rewrite',
+					'disabled'       => Util_Ui::sealing_disabled( 'browsercache.' ),
+					'control'        => 'checkbox',
 					'checkbox_label' => __( 'Rewrite <acronym title="Universal Resource Locator">URL</acronym> structure of objects', 'w3-total-cache' ),
-					'description' => __( 'Generate unique <acronym title="Universal Resource Indicator">URI</acronym> for each file protected from caching by browser.', 'w3-total-cache' ),
-					'label_class' => 'w3tc_single_column'
-				) );
+					'description'    => __( 'Generate unique <acronym title="Universal Resource Indicator">URI</acronym> for each file protected from caching by browser.', 'w3-total-cache' ),
+					'label_class'    => 'w3tc_single_column',
+				)
+			);
 			?>
 		</table>
 
-		<?php Util_Ui::button_config_save( 'browsercache_general' ) ?>
-		<?php Util_Ui::postbox_footer() ?>
+		<?php Util_Ui::button_config_save( 'browsercache_general' ); ?>
+		<?php Util_Ui::postbox_footer(); ?>
 
-		<?php Util_Ui::postbox_header( __( '<acronym title="Cascading Style Sheet">CSS</acronym> &amp; <acronym title="JavaScript">JS</acronym>', 'w3-total-cache' ), '', 'css_js' ) ?>
-		<p><?php _e( 'Specify browser cache policy for Cascading Style Sheets and JavaScript files.', 'w3-total-cache' ) ?></p>
+		<?php Util_Ui::postbox_header( __( '<acronym title="Cascading Style Sheet">CSS</acronym> &amp; <acronym title="JavaScript">JS</acronym>', 'w3-total-cache' ), '', 'css_js' ); ?>
```

Comment:
Use `esc_html__()`.

Reply:
Changed to wp_kses, etc. and added translations where applicable